frontend/app/forms/bank/form-list.component.js

```diff
@@ -217,6 +217,19 @@ export default class FormList extends React.Component {
                 "/static/forms/reply-to-statement/03_Petitioner_Reply-3.png"
               ]}
             />
+            <FillableForm
+              name="Petition for Expungement (Special Certificate from BCI)"
+              readyForUsers={true}
+              keywords="petition for expungement special certificate from bci"
+              shortDescription=""
```

Comment:
```suggestion
              shortDescription="The form for petitioning for an expungement when you have a special certificate from the BCI."
```

In the app, hover over this form in the form bank to see that description present.